packages/@angular/cli/blueprints/class/index.ts

```diff
@@ -1,3 +1,5 @@
+import * as appUtils from '../../utilities/app-utils';
```

Comment:
Instead of `import * as`, just import the function you need. This is a TypeScript file, not a CommonJS one.